src/components/Identity/createIdSyncs.js

```diff
@@ -90,3 +92,21 @@ export default ({ destinations, config, logger, cookieJar }) => {
     });
   }
 };
+
+const hasExpired = () => {
+  const nowInHours = Math.round(
+    convertTimes(MILLISECOND, HOUR, new Date().getTime())
+  );
+  const timestamp = parseInt(cookieJar.get(ID_SYNC_TIMESTAMP) || 0, 36);
+
+  return nowInHours > timestamp;
+};
+
+export default (config, logger, _cookieJar) => {
+  cookieJar = _cookieJar;
```

Comment:
I think we should pass cookieJar down through `createProcessor` and `hasExpired`. @wgamboa did what you did in his PR and I suggested he change it. Here's the reasoning: https://github.com/adobe/alloy/pull/127#discussion_r310269642

Reply:
I went back on forth on this, but I should go forth because you're right :) thanks.